packages/larva-patterns/modules/heading/heading.h3.js

```diff
@@ -2,7 +2,7 @@ const clonedeep = require( 'lodash.clonedeep' );
 
 const heading = clonedeep( require( './heading.prototype' ) );
 
-heading.heading_classes = 'a-font-secondary-l lrv-u-color-brand-primary';
```

Comment:
This and the above are also added in the project-level patterns in Indiewire, so this causes no change in functionality.